ethergo/submitter/submitter.go

```diff
@@ -63,6 +64,8 @@ type txSubmitterImpl struct {
 	// to prevent memory leaks, this has a buffer of 1.
 	// callers adding to this channel should not block.
 	retryNow chan bool
+	// gasBlockCache is used to cache the gas block for a given chain.
+	gasBlockCache map[int]*types.Header
```

Comment:
Maps are not concurrency safe for writes in golang

Reply:
Fixed in 48d19f083e854021cb28a77800b3f4b535ba2bf0

Reply:
LGTM